src/main/java/com/jagrosh/jmusicbot/commands/owner/BlacklistUserCmd.java

```diff
@@ -0,0 +1,60 @@
+package com.jagrosh.jmusicbot.commands.owner;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.commands.OwnerCommand;
+import com.jagrosh.jmusicbot.settings.Settings;
+import static com.jagrosh.jmusicbot.utils.OtherUtil.parseUserID;
+
+/**
+ *
+ * @author Omar Sanchez <omarsanchezdev@gmail.com>
+ */
+
+public class BlacklistUserCmd extends OwnerCommand
+{
+    public BlacklistUserCmd(Bot bot)
+    {
+        this.name = "blacklist";
+        this.help = "blocks or unblocks a user from using commands";
+        this.arguments = "<action>|<user>";
+        this.aliases = bot.getConfig().getAliases(this.name);
+        this.guildOnly = false;
+    }
+
+    @Override
+    protected void execute(CommandEvent event)
+    {
+        if(event.getArgs().isEmpty())
+        {
+            event.reply(event.getClient().getError()+" Please include a user");
```

Comment:
Please use `replyError`/`replyWarning`/`replySuccess` instead. Older JMusicBot code used to do this, as these utility methods haven't existed back then.

Make this change across all `reply`'s you've added as well.

```suggestion
            event.replyError("Please include a user");
```